ingestion_tools/scripts/common/finders.py

```diff
@@ -217,6 +265,22 @@ def find(
             return []
         return self._get_results(cls, config, metadata, **parent_objects)
 
+    @classmethod
+    def handle_for_metadata_file(
+        cls,
+        klass,
```

Comment:
what is klass, i'm not familiar with this notation?

Reply:
The `klass` was supposed to be the Importer class. I will document that in code as well. :)